apps/prairielearn/src/migrations/20240911213026_submission_embedding_store.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS submission_grading_context_embeddings (
+  id bigserial PRIMARY KEY,
+  embedding vector (1536) NOT NULL,
+  submission_id BIGINT NOT NULL REFERENCES submissions ON DELETE CASCADE ON UPDATE CASCADE,
+  submission_text text NOT NULL
+);
+
+CREATE INDEX IF NOT EXISTS grading_context_embeddings_submission_id_idx ON submission_grading_context_embeddings (submission_id);
```

Comment:
When possible, indexes should use the full table name:

```suggestion
CREATE INDEX IF NOT EXISTS submission_grading_context_embeddings_submission_id_idx ON submission_grading_context_embeddings (submission_id);
```

Reply:
updated